packages/core/webpack.config.js

```diff
@@ -58,17 +58,17 @@ module.exports = {
           context: path.resolve(__dirname, '../', 'web-components'),
         },
         {
-          from: 'components/**/*',
+          from: 'collection/components/**/*',
           to: './',
           context: path.resolve(__dirname, '../', 'web-components/dist'),
         },
         {
-          from: 'assets/**/*',
+          from: 'collection/assets/**/*',
           to: './',
           context: path.resolve(__dirname, '../', 'web-components/dist'),
         },
         {
-          from: 'img/**/*',
+          from: 'collection/img/**/*',
```

Comment:
I don't understand the change here, could you elaborate? What is collections and why do we need it now?

Reply:
This was an artifact from trying to fix my failing environment yesterday. I reverted these changes.

Reply:
It looks like stencil 3 re-organized the dist directory. On build it looks like, instead of everything being in web-components/dist it places them in web-components/dist/collection.

Reply:
This seemed wrong to me, so I kept looking. I needed to add this to the outputTargets in the stencil.config.ts and then everything built like it used to:

```
{
      type: 'dist-custom-elements'
    }
```

I have reverted the change to the webpack.config.js in core.